### PR TITLE
Editor: Improve Featured Image Dropzone Colours

### LINKED ISSUE #36239: Classic Editor: Drop zone needs additional fine-tuning for colors
The normal state has been mostly fixed by #36074, but the hover state (white text on a light pink background) fails AA contrast checks:

Screenshot 2019-09-13 at 16 27 27

I'm also still seeing the gridicon in black instead of white in the latest version of Chrome for Mac:

Screen Shot 2019-09-23 at 1 33 53 PM

To reproduce:

1) Add a new post or page in the Classic Editor
2) Open the Featured Image accordion menu and drag and drop an image over it
3) Note how the colors change when the image is dragged in and out of the drop zone

_Originally posted by @Aurorum in https://github.com/Automattic/wp-calypso/issues/31888#issuecomment-531389176_

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

This fixes two things:

1. the dropzone is using `--color-white`, that doesn't exist anymore as of #35949 I think. that's causing the gridicon to use an odd colour
2. the hover contrast was previously too low

#### Testing instructions

In the Classic Editor, you can access this feature by dragging an image over the document and over the dropzone in the sidebar.

**Current experience:**

_Hovering image over document_
Screenshot 2019-09-23 at 19 22 51

_Hovering image over dropzone_
Screenshot 2019-09-23 at 19 23 02

**Proposed experience:**

_Hovering image over document_
Screenshot 2019-09-23 at 19 23 15

_Hovering image over dropzone_
Screenshot 2019-09-23 at 19 23 25

Basically, the colour change still reflects that the image is being hovered over the dropzone, it just becomes darker instead of lighter.

Fixes #36239

### CODE REVIEW

**Comment by sixhours** [comment]
This looks and works much better for me!
